core/src/main/resources/messages/messages_es.yml

```diff
@@ -142,7 +142,7 @@ Chest-Location:
ng skills\n      &7&owith this amazing outfit!\n  Santa:\n    whole-equip: '&f&lwhole &4&lSanta &f&lsuit'\n    helmet-name: '&4&lSanta &f&lHelmet'\n    chestplate-name: '&4&lSanta &f&lChestplate'\n    leggings-name: '&4&lSanta &f&lLeggings'\n    boots-name: '&4&lSanta &f&lBoots'\n    Description: '&7&oBecome Santa and deliver presents!'\n  Frozen:\n    whole-equip: '&f&lwhole &b&lFrozen &f&lsuit'\n    helmet-name: '&b&lFrozen &f&lHelmet'\n    chestplate-name: '&b&lFrozen &f&lChestplate'\n    leggings-name: '&b&lFrozen &f&lLeggings'\n    boots-name: '&b&lFrozen &f&lBoots'\n    Description: |-\n      &7&oThis suit belongs to the\n      &7&oLord of the Frozen Lands!\n  Equip: '%prefix% &9You equipped %suitname%!'\n  Unequip: '%prefix% &9You unequipped %suitname%!'\n  Suit-Part-Lore: '&9Suits Part'\n  Whole-Equip-Lore: '&7&oEquip the whole suit!'\n  Must-Remove:\n    HELMET: '%prefix% &c&lYou must remove your helmet to equip a suit helmet!'\n    CHESTPLATE: '%prefix% &c&lYou must remove your chestplate to equip a suit chestplate!'\n    LEGGINGS: '%prefix% &c&lYou must remove your leggings to equip suit leggings!'\n    BOOTS: '%prefix% &c&lYou must remove your boots to equip suit boots!'\nEmotes:\n  Equip: '%prefix% &9You equipped %emotename%'\n  Unequip: '%prefix% &9You unequipped %emotename%'\n  Cry:\n    Name: '&9&lCry'\n    Description: '&7&oAre you really sad? :('\n  Angry:\n    Name: '&c&lAngry'\n    Description: '&7&oShow your rage to the other players!'\n  Happy:\n    Name: '&e&lHappy'\n    Description: '&7&oDon''t worry, be happy!'\n  Cheeky:\n    Name: '&6&lCheeky'\n    Description: '&7&oYou like being cheeky? Well, this emote is for you!'\n  Love:\n    Name: '&4&lLove'\n    Description: '&7&oYou have beautiful eyes, do you know that?'\n  DealWithIt:\n    Name: '&7&lDeal With it'\n    Description: '&7&oDo you feel like showing off? This emote is for you!'\n  Cool:\n    Name: '&9&lCool'\n    Description: '&7&oKeep cool man!'\n  Surprised:\n    Name: '&6&lSurprised'\n    Description: '&7&oOH LORD!!!'\n  Wink:\n    Name: '&a&lWink'\n    Description: '&7&oHaving fun? Let someone know you are in on the joke'\n  Must-Remove-Helmet: '%prefix% &c&lYou must remove your helmet to equip an emote!'\nEnabled-SelfMorphView: '%prefix% &9You enabled self view for morphs!'\nDisabled-SelfMorphView: '%prefix% &9You disabled self view for morphs!'\nEnabled-Gadgets: '%prefix% &9You enabled gadgets!'\nDisabled-Gadgets: '%prefix% &9You disabled gadgets!'\nNot-Allowed-From-Console: '&c&lThis can''t be executed from console!'\nWorld-Disabled: '%prefix% &c&lCosmetics are disabled in this world!'\nRegion-Disabled: '%prefix% &c&lCosmetics are disabled in this area!'\nRegion-Disabled-Category: '%prefix% &c&l%category% cosmetics are disabled in this\n  area!'\nDisabled-Command-Message: '%prefix% &c&lYou can''t use this command while cosmetics\n  are equipped!'\nNot-Allowed-In-Vanish: '%prefix% &c&lYou can''t use cosmetics while in vanish!'\nToo-Long: Too long, try again"
```

Comment:
@datatags Do you know why unapproved translations are in the Crowdin pull requests? This happens every time someone suggests a translation. Can you disable that? Only approved translations should be in the Crowdin pull requests, otherwise stuff like this happens.

From the Crowdin documentation: "All translated **and approved** files will be automatically pushed as a merge request ...".
Is it a bug? 🤔

Reply:
I've just turned on the setting "Export only approved translations," I guess it's off by default so you don't have to be so concerned about approving the latest translations all the time? Either way, should be fixed now 👍